repo-fetchers/README.md

```diff
@@ -48,7 +48,13 @@ This script fetches all repositories from a GitHub organization.
 ```
 
 #### Description
-This script fetches all repositories from the specified GitHub organization. The `GITHUB_TOKEN` environment variable must be set for authentication.
+This script fetches all repositories from the specified GitHub organization.
+
+##### Prerequisites:
+1. You must also have the [GitHub CLI](https://cli.github.com/) installed.
+2. You must either have run [`gh auth login`](https://cli.github.com/manual/gh_auth_login) or set the `GITHUB_TOKEN` environment variable for authentication.
+
+(**note**: if you use a GitHub installation rather than `github.com` you will have to `gh auth login --hostname github.mycompany.com`)
```

Comment:
```suggestion
(**Note**: if you use a GitHub installation rather than `github.com` you will have to `gh auth login --hostname github.mycompany.com`)
```